apps/core/scenes/results/Results.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { QueryRenderer, graphql } from '@kiwicom/margarita-relay';
+import { StyleSheet, Text } from '@kiwicom/universal-components';
+
+import type { ResultsList as ResultsListType } from './__generated__/ResultsList.graphql';
+import ResultsList from './ResultsList';
+import EmptyResults from './EmptyResults';
+
+type Props = {|
+  +travelFrom: string,
+  +travelTo: string,
+  +dateFrom: string,
+  +dateTo: string,
+|};
+
+type ResultsType = {|
+  +searchItineraries?: ResultsListType,
+|};
+
+export default class Results extends React.Component<Props> {
+  renderInner = (props: ResultsType) => {
+    const { searchItineraries } = props;
+    if (searchItineraries) {
```

Comment:
searchItineraries could be defined, but edges could be 0. This logic should be in the ResultsList component

Reply:
Right, moved condition into ResultsList.